src/iap/shared/components/carousel/carousel.component.scss

```
@@ -0,0 +1,94 @@
+@import '/src/scss/constants.scss';
+@import '/src/scss/mixins.scss';
```

You can try to create some alias `@scss` for these scss styles like for shared directory and try to write something like here:
```suggestion
@import '@scss/constants.scss';
@import '@scss/mixins.scss';
```
P.S. It's just a suggestion. I haven't used it myself yet, but you can try. :smile: